liveview-android/src/main/java/org/phoenixframework/liveview/domain/LiveViewCoordinator.kt

```diff
@@ -78,55 +80,80 @@ class LiveViewCoordinator(url: String) : ViewModel() {
         }
 
     private fun parseTemplate(originalRenderDom: String) {
-        val currentDom = mutableListOf<ComposableTreeNode>()
-        Log.e("dom", "=====================> \n originalRenderDom: $originalRenderDom")
-
-        val elements =
-            Jsoup
-                .parse(originalRenderDom)
-                .apply { ownerDocument()!!.outputSettings().prettyPrint(false) }
-                .body()
-                .children()
-
-        elements.forEach { element ->
-            Log.e("VM", "=====================> \n element: $element")
+        val currentDom = mutableListOf(ComposableTreeNode(ComposableNodeFactory.createEmptyNode()))
+        val parsedDocument = Document.parse(originalRenderDom)
+        doc.merge(
+            parsedDocument,
+            object : Document.Companion.Handler() {
+                override fun onHandle(
+                    context: Document,
+                    changeType: Document.Companion.ChangeType,
+                    nodeRef: NodeRef,
+                    parent: NodeRef?
+                ) {
+                    super.onHandle(context, changeType, nodeRef, parent)
+                    when (changeType) {
+                        Document.Companion.ChangeType.Change -> {
+                            println("Changed: ${context.getNodeString(nodeRef)}")
+                        }
+                        Document.Companion.ChangeType.Add -> {
+                            println("Added: ${context.getNodeString(nodeRef)}")
+                        }
+                        Document.Companion.ChangeType.Remove -> {
+                            println("Removed: ${context.getNodeString(nodeRef)}")
```

Comment:
This is just printing and not logging. Where does the `stdout` go when it is used on an Android device?